rmw_dds_common/include/rmw_dds_common/graph_cache.hpp

```diff
@@ -518,19 +527,28 @@ RMW_DDS_COMMON_PUBLIC
 std::ostream &
 operator<<(std::ostream & ostream, const GraphCache & topic_cache);
 
+/// Structure to represent the discovery data of a Participant.
 struct ParticipantInfo
 {
+  /// Discovery information of each Node created from a Participant.
   GraphCache::NodeEntitiesInfoSeq node_entities_info_seq;
+  /// Name of the enclave.
```

Comment:
@ahcorde @ivanpauno nit: perhaps we can link to ROS 2 security documentation?

Reply:
I would prefer not, links can easily get broken in the future.
The first result of googling "ros2 enclave" is the appropriate documentation.

Reply:
Fair enough.